pkgs/tools/security/browserpass/default.nix

```diff
@@ -46,7 +46,7 @@ buildGoModule rec {
 
   installPhase = ''
     make install
-
+    patchelf --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" $out/bin/browserpass
```

Comment:
Wondering if autopatchElfHook couldn't take care of that.

Reply:
works indeed